### ISSUE TITLE
Define Release Management for CAMARA

### ISSUE THREAD

**Issue by MarkusKuemmerle** [camaraproject/Governance#82]
- Release Cycle
- Coordination between Sub Projects
- Release Scope Management
- Alignment with Open Gateway (see documentation/MeetingMinutes/CAMARA Project - SteeringCommittee - 2023-07-13 - Open Gateway API Roadmap Presentation to CAMARA TSC.pdf)

**Comment by Bart-Ericsson** [comment]
I agree that the release cycle shall be agreed and scope defined. It should be a regular cycle (e.g. twice per year) where Sub Projects  should stick to if they want to bring improvements to the respective APIs. This shall be managed by the TSC in my view.
But I do not agree with alignment with Open Gateway, Open Gateway have their own release cycle and can bring API proposals to Camara at their own pace independent of the Camara cycle. We cannot have a dependency on external initiatives.

**Comment by hdamker** [comment]
We need to slice the elephant, most probably in different issues:

> * Release Cycle
Some sub project have already done their second or third release (but almost all alpha releases), some sub project haven't yet done a first one. CAMARA has to decide how often we want to make a "release" across the subprojects and how this is coordinated, e.g. with milestone during the release cycle. 

> * Coordination between Sub Projects

That includes the guidelines from Commonalities and Identity & Consent, and how they are applied and ensured within the sub projects (also: how are changes coming from these working groups are related to release cycles).

> * Release Scope Management

For existing and new APIs ... to be done within the sub projects by the sub project maintainers based on input from GSMA OPAG (OGW) and other sources. Documented until a milestone within the release cycle.

> * Alignment with Open Gateway (see documentation/MeetingMinutes/CAMARA Project - SteeringCommittee - 2023-07-13 - Open Gateway API Roadmap Presentation to CAMARA TSC.pdf)

That's is the point addressed by @HeleneVigue above in https://github.com/camaraproject/Governance/issues/82#issuecomment-1706939666. It is related to the three points above, but just one input for them.

**Comment by hdamker** [comment]
@HeleneVigue thanks for your input. We might be not able to discuss all points within the upcoming first TSC meeting, therefore some comments from my point of view here and requst to others to add also their comments:

> 
> 1. Use Open Gateway (OGW) Service template to provide inputs to CAMARA

We can't use just the Open Gateway template, but need our own in CAMARA for API proposals. CAMARA as an open project (see ProjectCharter) can get inputs and especially proposals for API definitions from different parties. 
My proposal would be to create a separate issue to discuss this point and then a pull request with proposed changes to the [API-proposal-template.md ](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-proposal-template.md) with some of the changes.

> 2. Discuss an agile interaction model for collaboration between OGW Product WS and CAMARA

That certainly needs discussion. I would prefer a more decoupled model, where both organization can work according their pace. E.g. I don't get why API proposals have to come in "drops" and will not just provided when they individually have reached a required maturity and support. That would avoid peaks of work in both organizations. New API proposals could be considered for a release if they were provided before a certain milestone for that release.

> 3. Invite CAMARA (EUC and TSC) to regular workshops, to provide insight on new API/technologies and hear market feedback

I consider this a good idea.

**Comment by jordonezlucena** [comment]
@shilpa-padgaonkar: should then close this issue and follow discussion in https://github.com/camaraproject/Commonalities/issues/67? Or what way forward you suggest?

**Comment by hdamker** [comment]
@jordonezlucena within this issue we have some more aspects beyond the more technical release management (like the working mode with OGW product team). And I would keep this issue as the reminder that his is currently an active topic within TSC discussions.
